tests/scripts/setup-pytest-env.sh

```diff
@@ -24,11 +24,14 @@ if [[ ! -z $CI_PYTEST_ADD_OPTIONS ]]; then
 else
     export PYTEST_ADDOPTS="-s -vv $PYTEST_ADDOPTS"
 fi
-set -u
+set -ux
 
 export TVM_PATH=`pwd`
 export PYTHONPATH="${TVM_PATH}/python"
 
+# TODO: Remove this line once https://github.com/apache/tvm/pull/12055 is merged into the Docker images
+python3 -m pip install --user pytest-rerunfailures==10.2
```

Comment:
maybe to reduce risk we should just update the ci images and turn this on?

Reply:
it's not a huge difference either way, but we've done this kind of thing before (with Jinja and `generate.py` and it turned out fine